WcaOnRails/app/views/incidents/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<% provide(:title, "Incident: #{@incident.name}") %>
+
+<%= render layout: "nav" do %>
+  <h1><%= yield(:title) %></h1>
+
+  <p>
+  <% @incident.tags_array.each do |t| %>
+    <%= render "incident_tag", tag: t %>
+  <% end %>
+  </p>
+  <p>
+  <% @incident.competitions.each do |c| %>
+    <%= render "competition_tag", comp: c %>
+  <% end %>
+  </p>
+
+  <% if @incident.status == "pending" %>
```

Comment:
Can we make a method for this? I'm always nervous about a typo in something like this and it just silently always returning false.